bin/isanetbelow60.py

```diff
@@ -1,9 +1,10 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 import os
 import requests
 import json
 import re
-import urllib2
+#import urllib2
+import urllib.request
 import sys
 import time
 #import ystockquote
```

Comment:
I feel we can just remove this commented line.  Yahoo is dead to you

Reply:
You're right, who needs them?